### PR TITLE
Tray Icon : Changed Unread tray icon in Windows .

### LINKED ISSUE #506: Tray icon when income new messages
---

- **Operating System**:
  - [x] Windows
  - [ ] Linux/Ubutnu
  - [ ] macOS
- **Clear steps to reproduce the issue**:
Normal icon and icon when incoming new message not strong different.
For example in Skype when new message in icon bright number and good see that incoming new message. Attached two screenshots for example.
- **Relevant error messages and/or screenshots**:
screenshot1
screenshot2

### PR DESCRIPTION
**Pull request by aryanshridhar**
Replaced the unread tray icon in Windows with a new icon as suggested by @timabbott <a href = 'https://github.com/zulip/zulip-desktop/issues/506#issuecomment-622043603'>here</a> .

<strong>Replaced Unread icon :</strong> <a href = 'https://github.com/zulip/zulip-desktop/blob/1d70f334882e2b2056482ce0ad64d23ba5d7c888/app/resources/tray/traywin.png'>here</a>

traylinuxunread

<strong>Working Icon in Tray-Bar :</strong>
zulipunread

Fixes #506

**You have tested this PR on:**
  - [x] Windows
  - [ ] Linux/Ubuntu
  - [ ] macOS

P.S : Its only applicable to <code>Windows</code> platform , Would love to extend the same icon to other Operating Systems as well :)

Would love an review @manavmehta .

### CODE REVIEW

**Comment by akashnimare** [comment]
LGTM. @andersk what do you think?

**Comment by aryanshridhar** [comment]
Hey , Should this be extended to other platforms as well ?

**Comment by timabbott** [comment]
lgtm visually.

File: app/renderer/js/tray.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ const iconPath = (): string => {
 	return APP_ICON + (process.platform === 'win32' ? 'win.ico' : 'macOSTemplate.png');
 };
 
+const winTrayIconPath = (): string => APP_ICON + 'win.png';
```
Comment (andersk) [inline app/renderer/js/tray.ts]: This filename and variable name make no sense for an icon that’s specifically for the unread case. Also, it should be in `.ico` format for Windows.

Comment (aryanshridhar) [reply app/renderer/js/tray.ts]: Oh Cool , Yeah it should have been in a <code>.ico</code> format , Will make necessary changes 👍 .
How about renaming <code>winTrayIconPath</code> to <code>winUnreadTrayIconPath</code> and similarly for the <code>.ico</code> file .

Comment (aryanshridhar) [reply app/renderer/js/tray.ts]: @andersk , Am I missing out something?

**Pull request merged**